CHANGELOG.md

```diff
@@ -12,6 +12,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Drop support for [Go 1.22]. (#6381, #6418)
 
+### Changed
+
+- ⚠️ Update `github.com/prometheus/client_golang` to `v1.21.1`, which changes the `NameValidationScheme` to `UTF8Validation`.
```

Comment:
Not sure about the emoji. We didn't do that for the `prometheus/common` upgrade.

Reply:
Want me to remove this? Thought it'd be easier to draw attention as it is under `Changed` heading and not something like `Breaking`.

Reply:
It's not a bad idea either, since this can be a confusing change. So let's keep it, unless someone objects.